src/library-authoring/component-info/ComponentManagement.tsx

```diff
@@ -0,0 +1,57 @@
+import { getConfig } from '@edx/frontend-platform';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Collapsible, Icon, Stack } from '@openedx/paragon';
+import { Tag } from '@openedx/paragon/icons';
+
+import { useLibraryBlockMetadata } from '../data/apiHooks';
+import StatusWidget from '../generic/status-widget';
+import messages from './messages';
+
+interface ComponentManagementProps {
+  usageKey: string;
+}
+const ComponentManagement = ({ usageKey }: ComponentManagementProps) => {
+  const intl = useIntl();
+  const { data: componentMetadata } = useLibraryBlockMetadata(usageKey);
+
+  if (!componentMetadata) {
+    return null;
+  }
+
+  return (
+    <Stack gap={3}>
+      <StatusWidget
+        {...componentMetadata}
+      />
+      {[true, 'true'].includes(getConfig().ENABLE_TAGGING_TAXONOMY_PAGES)
+        && (
+        <Collapsible
+          defaultOpen
+          title={(
+            <Stack gap={1} direction="horizontal">
+              <Icon src={Tag} />
+              {intl.formatMessage(messages.manageTabTagsTitle)}
+            </Stack>
+          )}
+          className="border-0"
+        >
+          Tags placeholder
+        </Collapsible>
+        )}
+      <Collapsible
+        defaultOpen
+        title={(
+          <Stack gap={1} direction="horizontal">
+            <Icon src={Tag} />
+            {intl.formatMessage(messages.manageTabCollectionsTitle)}
+          </Stack>
+        )}
+        className="border-0"
+      >
+        Collections placeholder
```

Comment:
Is showing the component's Collections out of scope here? Thanks to your suggestions, we should have data available once [edx-platform#35469](https://github.com/openedx/edx-platform/pull/35469/files#diff-8c7bc1adff4ab3b98ae2dfca83b4d50407bb2be71d45de93e504552424703fe2R248-R252) merges.

Reply:
I will create a follow up tasks to handle the collections.